src/app/pages/signing/signing.component.ts

```diff
@@ -8,9 +8,44 @@ import { Component, OnInit } from '@angular/core';
 
 export class SigningComponent implements OnInit {
   public signingBoxText = '';
+  // fonts and colors
+  public currentFont = 'Arial';
+  public backgroundColor = '#ffffff';
+  public textColor = '#000000';
+
+  selectFont(font: string) {
+    this.currentFont = font;
+  }
```

Comment:
Add blank line after function
```suggestion
  }
  
```